libs/cde-visualization/src/lib/components/violin/violin.component.html

```diff
@@ -16,7 +16,16 @@
     [cdkConnectedOverlayOpen]="infoOpen"
     [cdkConnectedOverlayOffsetX]="5"
   >
-    <div>Violin plot</div>
+    <hra-tooltip-card
+      small
+      [content]="[
```

Comment:
Move content into a readonly variable on the component class and use that instead of embedding complex objects in the html